sonar-scanner-engine/src/main/java/org/sonar/batch/scan/ProjectBuildersExecutor.java

```diff
@@ -41,14 +42,18 @@ public ProjectBuildersExecutor() {
   }
 
   public void execute(ProjectReactor reactor) {
-    if (projectBuilders.length > 0) {
-      Profiler profiler = Profiler.create(LOG).startInfo("Execute project builders");
-      ProjectBuilderContext context = new ProjectBuilderContext(reactor);
-
-      for (ProjectBuilder projectBuilder : projectBuilders) {
-        projectBuilder.build(context);
+    try {
+      if (projectBuilders.length > 0) {
+        Profiler profiler = Profiler.create(LOG).startInfo("Execute project builders");
+        ProjectBuilderContext context = new ProjectBuilderContext(reactor);
+
+        for (ProjectBuilder projectBuilder : projectBuilders) {
+          projectBuilder.build(context);
+        }
+        profiler.stopInfo();
       }
-      profiler.stopInfo();
+    } catch (Exception e) {
```

Comment:
Why don't you make the try..catch surround projectBuilder.build(context); so that you can log exactly what was the ProjectBuilder involved?